docs/source/reference/faq.rst

```diff
@@ -79,17 +79,22 @@ To avoid rerunning the ``setup`` commands, pass the ``--no-setup`` flag to ``sky
 (Advanced) How to make SkyPilot use all global regions?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, SkyPilot only supports the US regions on different clouds for convenience. If you want to utilize all global regions, please run the following command:
+By default, SkyPilot only supports the US regions on different clouds (except for AWS) for convenience. If you want to utilize all global regions, please run the following command:
 
 .. code-block:: bash
 
-  mkdir -p ~/.sky/catalogs/v4
-  cd ~/.sky/catalogs/v4
+  version=$(python -c 'import sky; print(sky.clouds.service_catalog.constants.CATALOG_SCHEMA_VERSION)')
+  mkdir -p ~/.sky/catalogs/${version}
+  cd ~/.sky/catalogs/${version}
   # Fetch all regions for AWS
-  python -m sky.clouds.service_catalog.data_fetchers.fetch_aws --all-regions
+  # Note: SkyPilot already uses all global regions for AWS, and will periodically update the price
+  # manually running this command will disable the automatic update.
+  # python -m sky.clouds.service_catalog.data_fetchers.fetch_aws --all-regions
```

Comment:
Do we want to leave this out? Or, say some reason why users may want to run this?

Reply:
Good point! Removed it for now.